Index.html

```diff
@@ -77,5 +77,170 @@ <h1>I'm Kerlos Baha.<br />Glad to see <br />you!</h1>
       </section>
       <span class="shape"> </span>
     </main>
+    <div class="space"></div>
+
+    <!-- first work page -->
+
+    <section class="repeated-section">
+      <img class="repeated-img" src="/picture/Snapshoot Portfolio.png" alt="" />
+      <p class="repeated-p1">Tonic</p>
+      <ul class="repeated-ul">
+        <li class="repeated-li1">CANOPY</li>
+        <li class="repeated-li2">Back End Dev</li>
+        <li class="repeated-li3">2015</li>
+      </ul>
+      <p class="repeated-p2">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+      </p>
+      <ul class="repeated-ul2">
+        <li class="repeated-li4">HTML</li>
+        <li class="repeated-li5">CSS</li>
+        <li class="repeated-li6">JavaScript</li>
+      </ul>
+      <button class="see-button">See project</button>
+    </section>
+
+    <div class="space-2"></div>
+    <!-- second work page -->
+    <section class="repeated-section-2">
+      <img
+        class="repeated-img"
+        src="/picture/Snapshoot Portfolio2.png"
+        alt=""
+      />
+      <p class="repeated-p1">
+        Multi-Post <br />
+        Stories
+      </p>
+      <ul class="repeated-ul">
+        <li class="repeated-li1">CANOPY</li>
+        <li class="repeated-li2">Back End Dev</li>
+        <li class="repeated-li3">2015</li>
+      </ul>
+      <p class="repeated-p2">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+      </p>
+      <ul class="repeated-ul2">
+        <li class="repeated-li4">HTML</li>
+        <li class="repeated-li5">CSS</li>
+        <li class="repeated-li6">JavaScript</li>
+      </ul>
+      <button class="see-button">See project</button>
+    </section>
+
+    <!-- third page -->
+    <div class="space-2"></div>
+    <section class="repeated-section">
+      <img
+        class="repeated-img"
+        src="/picture/Snapshoot Portfolio3.png"
+        alt=""
+      />
+      <p class="repeated-p1">Tonic</p>
+      <ul class="repeated-ul">
+        <li class="repeated-li1">CANOPY</li>
+        <li class="repeated-li2">Back End Dev</li>
+        <li class="repeated-li3">2015</li>
+      </ul>
+      <p class="repeated-p2">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+      </p>
+      <ul class="repeated-ul2">
+        <li class="repeated-li4">HTML</li>
+        <li class="repeated-li5">CSS</li>
+        <li class="repeated-li6">JavaScript</li>
+      </ul>
+      <button class="see-button">See project</button>
+    </section>
+    <!-- four page -->
+    <div class="space-2"></div>
```

Comment:
Instead of using a div to create space between each card, it would be much better to wrap all your cards in a `<div>` tag and use grid styling to add gaps in between each card🙏. This is a much better implementation and is also one of the project requirements.

<img width="745" alt="image" src="https://user-images.githubusercontent.com/37116322/227352248-7b92fca3-7a90-46c0-a3ae-fd1c6ebeffbd.png">

Please update your works section to use CSS grid to style the layout of all the cards and CSS flexbox to style the layout of each individual card🎨.